README.md

```diff
@@ -92,6 +92,8 @@ static partial class Log
 }
 ```
 
+TODO: TEST - When not provided, source generator will produce diagnostic pointing to message being required.
+
```

Comment:
```suggestion
```
TODO: TEST - When not provided, source generator will produce diagnostic pointing to message being required.